test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -37,7 +37,7 @@ def to_numpy(tensor):
     inputs = list(map(to_numpy, input))
     outputs = list(map(to_numpy, output))
 
-    ort_inputs = dict((ort_sess.get_inputs()[i].name, input) for i, input in enumerate(inputs))
+    ort_inputs = {inp.name: inputs[i] for i, inp in enumerate(ort_sess.get_inputs())}
```

Comment:
I mentioned this in the description:
> This includes a minor change to `ort_test_with_input` to only select the inputs that are specified in the session, instead of the PyTorch inputs. This allows support for the `out` optional argument, since it is not actually used as an input in the session -- instead it's type is used to cast the output appropriately.